src/Trustline.ts

```diff
@@ -83,6 +84,7 @@ export class Trustline {
       gasLimit,
       gasPrice
     } = options
+    const isFrozen = options.isFrozen
```

Comment:
Shouldn't this be moved into the `const {...}` block above?

Reply:
yes